src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const properties = sourceString.split(';');
```

Comment:
Using 'split' directly on the 'sourceString' might include empty strings if there are consecutive semicolons. It would be beneficial to filter out empty declarations to ensure the object does not contain empty keys.